### ISSUE TITLE
XSLT: xsl:param 'content-handler' is not declared for template 't:get-unit-declarations'

### ISSUE THREAD

**Issue by vnesterovsky** [DeltaXML/vscode-xslt-tokenizer#64]
Still tunnel parameters are warned about:

```xslt
    <xsl:call-template name="t:get-unit-declarations">
      <xsl:with-param name="content-handler" tunnel="yes" as="element()">
        <t:unit-declaration/>
      </xsl:with-param>
    </xsl:call-template>
```

This is similar to issue #57

**Comment by pgfearo** [comment]
Ok thanks, I had thought this was fixed - I'll investigate further.

**Comment by pgfearo** [comment]
Ok, I have a hunch that this issue was not fixed by the #57 fix because `xsl:with-param` was a self closing tag in that case so the code path was different

**Comment by pgfearo** [comment]
Fixed
